css/style.css

```diff
@@ -78,6 +82,116 @@ footer {
 /* ADDITIONAL STYLES
 /*******************************************/
 
+h1 { 
+  font-size: 24px;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h2 {
+  font-size: 18px;
+}
+.txt-white  {
+  color: #fff;
+}
+
+.txt-yellow{
+  color: yellow;
+}
 
+.top-bar {
+  padding-top: 20px;
+  padding-left: 20px;
+  font-weight: 800;
+}
+
+.tag_line {
+  text-align: center;
+  padding: 30px;
+  font-weight: 500;
+}
+
+.shop1{
+  padding: 20px 20px 200px 20px;
+  text-align: center;
+}
+
+
+#button1 {
+  width: 300px;
+  background-color: #f9e42e;
+  color: #033048;
+  font-weight: 800;
+  font-size: 14px;
+  border-radius: 5px;
+  border: none;
+  padding: 20px;
+  margin: 0 auto;
+}
 
+.benefits{
+  background-color: #f0efef; 
+  color: #606161;
+  padding: 30px 20px;
+  font-size: 13px;
+  font-weight: 300;
+}
+
+.benefits h2 {
+  font-weight: 500;
+  color: #033048;
+}
+
+
+.benefits li:before {
```

Comment:
@Uliana202 Very resourceful solution to get the bullet points you wanted! Awesome job here.